### ISSUE TITLE
Viewer can change the position of icon

### ISSUE THREAD

**Issue by amolbhawari** [cloudskew/cloudskew#53]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**
If possible, add screenshots and/or screen recordings which capture the issue. These greatly help investigations.

**Errors in browser console**
If you observe any errors in the browser console, please paste those here.

**Desktop (please complete the following information):**
 - Browser [e.g. chrome, safari, firefox]
 - OS: [e.g. windows10, macOS]

**Additional details**
Add any other details about the problem here.

**Comment by mithunshanbhag** [comment]
Yes, it is intentional (to allow the viewer to interact & experiment with the diagram. This does not modify the original diagram.  

Refreshing the page will bring diagram back to original state.

**Comment by mithunshanbhag** [comment]
[update 6/5/2020] 

1. We're now restricting diagram viewer operation to just panning mode. Symbols cannot be selected or edited.

2. Context menu items still need to be disabled. 

Fix will be available in production in a couple of days.

**Comment by mithunshanbhag** [comment]
[update 6/8/2020] Context menu items now disabled in diagram viewer. 

Fix is currently in pre-production environment. Will be pushed to production after testing (in a day or two).
